assets/extensions/examples/online-extensions.css

```diff
@@ -0,0 +1,6 @@
+/*
+  Customize the logo in the homepage header
+*/
+.navbar-pf-alt #header-logo {
```

Comment:
yep i have the changes locally to remove the nesting

On Wed, Feb 10, 2016 at 3:54 PM, Ben Petersen <notifications@github.com>
wrote:

> In assets/extensions/examples/online-extensions.css
> <https://github.com/openshift/origin/pull/7169#discussion_r52523211>:
>
> > @@ -0,0 +1,6 @@
> > +/*
> > +  Customize the logo in the homepage header
> > +*/
> > +.navbar-pf-alt #header-logo {
>
> +1 to less nested styles when possible
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7169/files#r52523211>.
>